Gemfile

```diff
@@ -2,3 +2,7 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in redis-objects.gemspec
 gemspec
+
+group :development, :test do
+  gem 'redis-namespace', git: 'https://github.com/resque/redis-namespace'
```

Comment:
@ota42y why did you move `redis-namespace` from the `gemspec` file to the `Gemfile`?